plugin-server/src/types.ts

```diff
@@ -633,13 +639,15 @@ export interface Team {
     api_token: string
     slack_incoming_webhook: string | null
     session_recording_opt_in: boolean
-    person_processing_opt_out?: boolean
+    person_processing_opt_out: boolean | null
     heatmaps_opt_in: boolean | null
     ingested_event: boolean
     person_display_name_properties: string[] | null
     test_account_filters:
         | (EventPropertyFilter | PersonPropertyFilter | ElementPropertyFilter | CohortPropertyFilter)[]
         | null
+    cookieless_server_hash_mode: CookielessServerHashMode | null
+    timezone: string
```

Comment:
I assume this `timezone` is unrelated for now but it'll be used eventually?

Reply:
Yeah, exactly, it didn't feel like splitting that up further would be a good use of time